src/Serilog/Core/ForContextExtension.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Serilog.Events;
+
+namespace Serilog
+{
+    /// <summary>
+    /// Extension method 'ForContext' for ILogger.
+    /// </summary>
+    public static class ForContextExtension
```

Comment:
It's possible we'll add more extensions on `ILogger` in the future; shall we name this one `LoggerExtensions`?

Reply:
Sure - my idea here was to create one extension class for each method name/overload and create more classes as more "types" of extension methods was introduces. But, it makes sense to have one class to start with and break it out to different classes when the need arises.